sdcm/cluster.py

```diff
@@ -4338,6 +4346,24 @@ def create_cluster_manager(self, cluster_name: str, manager_tool=None, host_ip=N
             host_ip = self.nodes[0].ip_address
         return manager_tool.add_cluster(name=cluster_name, host=host_ip, auth_token=self.scylla_manager_auth_token)
 
+    def is_ebs_volumes_attached(self) -> bool:
+        """check data device type usage
+
+        check if scylla cluster uses the
+        ebs volumes for instance on aws cloud
+
+        :returns: if ebs volumes attached return true
+        :rtype: {bool}
+        """
+        backend = self.params.get('cluster_backend')
+        data_device_type = self.params.get("data_device")
+
+        if backend == "aws" and data_device_type == "attached":
+            self.log.info("EBS volumes attached")
```

Comment:
Let's don't log it here, if you want to log such a thing please do it on SCTConfiguration or setup level.

Reply:
removed